test/fixtures/section-heading-first-fail.html

```diff
@@ -0,0 +1,3 @@
+<section>
+    <p>Hello, world.</p>
+</section>
```

Comment:
I see. In that case, grouping them into one test confused me; I thought the unit under test was broader (page) as opposed to a tighter unit (check). In that case I take it back; you can feel free to remove that test. Thanks for discussing it.